tests/Satooshi/ProjectTestCase.php

```diff
@@ -7,7 +7,7 @@ class ProjectTestCase extends \PHPUnit_Framework_TestCase
     protected function setUpDir($projectDir)
     {
         $this->rootDir       = realpath($projectDir . '/prj');
-        $this->srcDir        = $this->rootDir . '/files';
+        $this->srcDir        = realpath($this->rootDir . '/files');
```

Comment:
Realpath fix them. Project dir as well.

In tests/Satooshi/ProjectTestCase.php
<https://github.com/satooshi/php-coveralls/pull/177#discussion_r48685811>:

> @@ -7,7 +7,7 @@ class ProjectTestCase extends \PHPUnit_Framework_TestCase
>      protected function setUpDir($projectDir)
>      {
>          $this->rootDir       = realpath($projectDir . '/prj');
> -        $this->srcDir        = $this->rootDir . '/files';
> +        $this->srcDir        = realpath($this->rootDir . '/files');

Are '/files' and '/prj' OK? Shouldn't we use DIRECTORY_SEPARATOR here and
there?

—
Reply to this email directly or view it on GitHub
<https://github.com/satooshi/php-coveralls/pull/177/files#r48685811>.